README.md

```diff
@@ -1 +1,109 @@
-# webpack-setup-basics
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MD Seafoods
+
+This website fetches seafoods meals gotten from an API [MealsDb](https://themealdb.com/browse.php?b=f) and display on the homepage.
+One can like a food by clicking on the heart icon and comment on a specific food by clicking on the comment button to have access to the comment form and place your comment in an instant.
+
+### ScreenShoot - Mobile screens
+
+![image](https://user-images.githubusercontent.com/50941074/147270738-aa0e9cb1-8fb3-4e16-9df1-614d636f1e05.png)
+
+### ScreenShoot - Desktop screens
+
+![image](https://user-images.githubusercontent.com/50941074/147270429-b2826c00-19b3-4fbb-bde1-a5d234117dd6.png)
+
+## Built With
+
+- HTML
+- CSS 
+- JavaScript ES6
+- Tailwind CSS 
+- Webpack-5
+- Jest
+
+## Live Demo
+
+[Live Demo](https://seafoods-app.netlify.app/)
+
+## Video Documentation
+[Video](https://drive.google.com/file/d/1yJMniajNASvKvGaUk6lF7yoetIP4ZV-C/view?usp=sharing)
+
```

Comment:
- [ ] You have really done a great job on the presentation but one thing missing. Please highlight a piece of the code you find interesting or are proud of making.

Reply:
Oooh, ok, bro. We'll definitely redo and mention that. 👍